src/SFA.DAS.AssessorService.Application.Api/Validators/EpaOrganisationValidator.cs

```diff
@@ -65,6 +72,20 @@ public string CheckUkprnIsValid(long? ukprn)
             return isValid ? string.Empty : FormatErrorMessage(EpaOrganisationValidatorMessageName.UkprnIsInvalid);
         }
 
+        public string CheckOrganisationNameNotUsed(string name)
+        {
+            return _registerRepository.EpaOrganisationAlreadyUsingName(name, string.Empty).Result 
+                ? ErrorMessageOrganisationNameAlreadyPresent : 
+                string.Empty;
+        }
+
+        public string CheckOrganisationNameNotUsedForOtherOrganisations(string name, string organisationIdToIgnore)
+        {
+            return _registerRepository.EpaOrganisationAlreadyUsingName(name, organisationIdToIgnore).Result 
+                ? ErrorMessageOrganisationNameAlreadyPresent : 
+                string.Empty;
+        }
+
```

Comment:
Would if possible consider using await ? think result is synchronous - see https://stackoverflow.com/questions/24623120/await-on-a-completed-task-same-as-task-result